<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Presentation</title>

		<meta name="description" content="Brandeis 2017 Talk">
		<meta name="author" content="Michael J. Harms">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">

        <link rel="stylesheet" href="css/bootstrap.min.css">
		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/white.css" id="theme">

		<!-- Code syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>

		<link rel="stylesheet" href="presentation-data/css/override.css">
	</head>

	<body>

		<div class="reveal">
			<!-- Any section element inside of this container is a slide -->
			<div class="slides">

                <section>
                    <h4>Harms Lab</h4>
                    <h4>University of Oregon</h4>
                </section>

                <section>
                    <p>How does evolution assemble such an <em>amazing</em> diversity of proteins?</p>
                    <div class="row">
                        <div class="col-xs-4">
                            <img class="img-responsive" src="presentation-data/img/protein-diversity/0.png" />
                        </div>
                        <div class="col-xs-4">
                            <img class="img-responsive" src="presentation-data/img/protein-diversity/1.png" />
                        </div>
                        <div class="col-xs-4">
                            <img class="img-responsive" src="presentation-data/img/protein-diversity/2.png" />
                        </div>
                    </div>

                    <div class="row">
                        <div class="col-xs-4">
                            <img class="img-responsive" src="presentation-data/img/protein-diversity/3.png" />
                        </div>
                        <div class="col-xs-4">
                            <img class="img-responsive" src="presentation-data/img/protein-diversity/4.png" />
                        </div>
                        <div class="col-xs-4">
                            <img class="img-responsive" src="presentation-data/img/protein-diversity/5.png" />
                        </div>
                    </div>
                </section>
            
                <section> 
                    <h4>Example: Why does Europe have the borders it does?</h4>

                    <video data-autoplay height="60%" width="60%">
                        <source src="https://googledrive.com/host/0BwbyvsegsV0db3JGTHdCYVBXVDQ" />
                    </video>
                    <br/>
                    <small><p align="right">
                        <a href="https://www.youtube.com/watch?v=uxDyJ_6N-6A">
                            adapted from video by MrOwnerandPwner
                        </a>
                    </p></small>
                </section>                   

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row fragment">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_0.png" height="90%" width="90%"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_1.png" height="90%" width="90%"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_2.png" height="90%" width="90%"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_3.png" height="90%" width="90%"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_4.png" height="90%" width="90%"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_5.png" height="90%" width="90%"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>We use ancestral sequence reconstruction to trace the evolution of proteins</h4>
                    <div class="row">
                        <img class="img-responsive" src="presentation-data/img/asr-overview/asr_6.png" height="90%" width="90%"/>
                    </div>
                </section>
    
                <section>
                    <h4>Some Questions:</h4>
                    <ul>
                        <li>How does this blind process put together integrated, complicated machines?</li>
                        <li class="fragment">Are there common mechanisms of evolutionary change at the molecular level (and what are they)?</li>
                        <li class="fragment">Is protein evolution predictable?</li>
                        <li class="fragment">Practically: how can we effectively navigate sequence space to new protein functions?</li>
                    </ul>
                </section>

                <section data-transition="fade">
                    <h4>(Partial) answers to these questions lie at the interface of biochemistry and evolution</h4><br/>
                    <img src="presentation-data/img/biochem-evolution_0.png" height="70%" width="70%" />
                </section>

                <section data-transition="fade">
                    <h4>(Partial) answers to these questions lie at the interface of biochemistry and evolution</h4><br/>
                    <img src="presentation-data/img/biochem-evolution_1.png" height="70%" width="70%" />
                </section>

                <section data-transition="fade">
                    <h4>(Partial) answers to these questions lie at the interface of biochemistry and evolution</h4><br/>
                    <img src="presentation-data/img/biochem-evolution_2.png" height="70%" width="70%" />
                </section>

                <section>
                    <h4>How should we think about this relationship?</h4>
                </section>

                <section data-transition="fade">
                    <p><em>Sequence space</em>: how does the word <span style="color:rgb(250,157,0);">RAGE</span> evolve into <span style="color:rgb(164,0,199);">COST</span>?</p> 
                    <img src="presentation-data/img/sequence-space_0.png"/>
                    <p style="color:white;">English shapes possible trajectories</p>
                    <small>Smith JM (1970), Nature (<a href="doi:10.1038/225563a0">doi:10.1038/225563a0</a>)</small>
                </section>

                <section data-transition="fade">
                    <p><em>Sequence space</em>: how does the word <span style="color:rgb(250,157,0);">RAGE</span> evolve into <span style="color:rgb(164,0,199);">COST</span>?</p> 
                    <img src="presentation-data/img/sequence-space_1.png"/>
                    <p style="color:white;">English shapes possible trajectories</p>
                    <small>Smith JM (1970), Nature (<a href="doi:10.1038/225563a0">doi:10.1038/225563a0</a>)</small>
                </section>
    
                <section data-transition="fade">
                    <p><em>Sequence space</em>: how does the word <span style="color:rgb(250,157,0);">RAGE</span> evolve into <span style="color:rgb(164,0,199);">COST</span>?</p> 
                    <img src="presentation-data/img/sequence-space_2.png"/>
                    <p>English shapes possible trajectories</p>
                    <small>Smith JM (1970), Nature (<a href="doi:10.1038/225563a0">doi:10.1038/225563a0</a>)</small>
                </section>

                <section data-transition="fade">
                    <p>Biochemistry determines the distribution of function in sequence space</p><br/>
                    <img src="presentation-data/img/biochem-evolution_2.png" height="70%" width="70%" />
                    <p>Evolutionary processes determine moves through the space</p>
                </section>

                <section>
                    <p>Lab approaches:</p>
                    <ul>
                        <li class="fragment">Trace historical changes in protein function.</p>
                        <li class="fragment">Study sequence space itself.</p>
                    </ul>
                </section>

                <section>
                    <h4>Today: describe two projects</h4>
                    <div class="row">
                        <div class="col-xs-2">
                            <img src="presentation-data/img/people/luke-wheeler_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-10"><p>Luke Wheeler: Promiscuity and specificity in the evolution of S100 proteins</p></div>
                    </div>

                    <div class="row">
                        <div class="col-xs-2">
                            <img src="presentation-data/img/people/zach-sailer_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-10"><p>Zach Sailer: High-order epistasis in genotype-phenotype maps</p></div>
                    </div>

                </section>

                <section>
                    <p>S100 proteins are small, $Ca^{2+}$ binding proteins: the lab workhorses!</p>
                </section>

                <section data-transition="fade">
                    <h4>S100 proteins bind short <span style="color:red;">peptide regions</span> of target proteins.</h4>
                    <div class="col-xs-12">
                        <img src="presentation-data/img/protein-structure_0.png" class="img-responsive"/>
                    </div>
                    <small align="center">PDB: 1K9P, 2JTT</small><br/>
                    <small align="center">Otterbein et al. (2002) Structure 10: 557-567</small>
                    <small align="center">Lee et al. (2008) Biochemistry 47: 10921-10932</small>
                </section>

                <section data-transition="fade">
                    <h4>S100 proteins bind short <span style="color:red;">peptide regions</span> of target proteins.</h4>
                    <div class="col-xs-12">
                        <img src="presentation-data/img/protein-structure_1.png" class="img-responsive"/>
                    </div>
                    <small align="center">PDB: 1K9P, 2JTT</small><br/>
                    <small align="center">Otterbein et al. (2002) Structure 10: 557-567</small>
                    <small align="center">Lee et al. (2008) Biochemistry 47: 10921-10932</small>
                </section>
               
                <section data-transition="slide">
                    <p>Peptide binding occurs in a $Ca^{2+}$-dependent manner</p>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/luke/ca-dep-itc.png" height="65%" width="65%" class="img-responsive"/>
                    </div>
                </section>
            
                <section>
                    <h4>The S100 protein family has expanded radically across the vertebrates</h4>
                    <div class="row">
                        <div class="col-xs-6">
                            <img src="presentation-data/img/oikopleura.jpg" height="86%" width="86%" />
                        </div>
                        <div class="col-xs-6">
                            <img src="presentation-data/img/snowden.jpg" height="100%" width="100%" />
                        </div>
                    </div>
                    <div class="row fragment">
                        <div class="col-xs-6">
                            <h3>2</h3>
                        </div>
                        <div class="col-xs-6">
                            <h3>30</h3>
                        </div>
                    </div>
                    <div class="row">
                        <div class="col-xs-6">
                            <small><a href="http://jellizone.com">jellizone.com</a></small>
                        </div>
                        <div class="col-xs-6">
                            <small><a href="http://theguardian.com">theguardian.com</a></small>
                        </div>
                    </div>
                    
                </section>

                <section>
                    <h4>Different S100 proteins have evolved a huge variety of functions:</h4>
                    <ul class="fragment">
                        <li>Regulation of cell motility and proliferation</li>
                        <li>Metal transport/sequestration</li>
                        <li>Direct antimicrobial activity</li>
                        <li>Pro-inflammatory signaling</li>
                        <li>Substrate transport</li>
                    </ul>
                </section>

                <section>
                    <h4>S100A5 and S100A6 have different functions</h4>
                    <ul>
                        <li class="fragment"><h4>S100A6 promotes cell motility and proliferation</h4>
                            <ul>
                                <li>Inhibits p53 tetramerization</li>
                                <li>Binds to proteins (annexins, calponin, tropomyosin), triggering cell motility</li>
                                <li>Is a marker for metastasis and malignancy</li><br/>
                            </ul>
                        <li class="fragment"><h4>S100A5 is involved in olfaction</h4>
                            <ul>
                                <li>One identified target (a $Na/Ca^{2+}$ transporter)</li>
                                <li>Massive increase in expression in olfactory sensory neurons upon olfactory stimulation</li>
                            </ul>
                        </li>
                    </ul>
                </section>

                <section>
                    <p><span style="color:rgb(164,0,199);">S100A5</span> and <span style="color:rgb(250,157,0);">S100A6</span> arose by gene duplication in the ancestor of amniotes</p>
                    <img src="presentation-data/img/phylogenetic-trees.png" height="100%" width="100%"/>
                </section>

                <section data-transition="slide">
                    <h4>S100A5 and S100A6 bind to different targets</h4>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/modern-peptide.png" height="75%" width="75%" class="img-responsive"/>
                    </div>
                    <small>Liriano (2012), Streicher et al. (2009) Biochemistry 48(12):2788</small>
                </section>

                <section>
                    <h4>Modern orthologs share common binding specificity</h4>
                    <img src="presentation-data/img/new-tree_no-anc.png" height="70%" width="70%"/>  

                </section>

                <section>
                    <h4>How did their target specficity evolve?</h4>
                    <ul class="fragment">
                        <li>What changed with respect to specificity?</li>
                        <li>What mutations were involved?</li>
                        <li>How did these mutations exert their effect?</li>
                    </ul>
                </section>
                 

                <section>
                    <p>We reconstructed ancA5/A6, the last common ancestor of S100A5 and S100A6</p>
                    <img src="presentation-data/img/phylogenetic-trees.png" height="100%" width="100%"/>
                </section>
        
                <section>
                    <p>Ancestor folds and responds to $Ca^{2+}$</p>
                    <div class="row">
                        <div class="col-xs-6"><img src="presentation-data/img/hA5-cd.png" class="img-responsive" /></div>
                        <div class="col-xs-6"><img src="presentation-data/img/ancA5-cd.png" class="img-responsive" /></div>
                    </div>
                    <p class="fragment">Peptide target binding is strictly $Ca^{2+}$-dependent</p>
                </section>   

                <section data-transition="fade">
                    <h4>What was the ancestral binding specificity?</h4><br/>
                    <div class="row">
                        <div class="col-xs-2"></div>
                        <div class="col-xs-8">
                            <img src="presentation-data/img/new-tree_no-anc.png" class="img-responsive">
                        </div>
                        <div class="col-xs-2"></div>
                    </div>
                </section>
                <section data-transition="fade">
                    <h4>Ancestor (apparently) binds to more targets than modern S100A5 and S100A6</h4>
                    <div class="row">
                        <div class="col-xs-2"></div>
                        <div class="col-xs-8">
                            <img src="presentation-data/img/new-tree.png" class="img-responsive">
                        </div>
                        <div class="col-xs-2"></div>
                    </div>
                </section>

                <section>
                    <h4>Maybe this is an example of a protein that went from promiscuity to specificity?</h4>
                    <ul class="fragment">
                        <li>New interactions often start as promiscuous interactions that are refined</li>
                        <li>Subfunctionalization of a promiscuous ancestor could lead to two more specific descendences</li>
                        <li>Proteins need fewer specificty "rules" to "parse" less chemically complex past environments</li>
                    </ul>
                </section>

                <section>
                    <p>But is this really an example of ancestral promiscuity?</p>
                    <p class="fragment">Just how should we measure promiscuity, anyway?</p>
                </section>

                <section data-transition="fade">
                    <h4>Measuring changes in ancestral binding requires an unbiased sample of targets</h4>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/binding-venn-0.png" height="75%" width="75%" class="img-responsive"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>Measuring changes in ancestral binding requires an unbiased sample of targets</h4>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/binding-venn-1.png" height="75%" width="75%" class="img-responsive"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>Measuring changes in ancestral binding requires an unbiased sample of targets</h4>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/binding-venn-2.png" height="75%" width="75%" class="img-responsive"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>Measuring changes in ancestral binding requires an unbiased sample of targets</h4>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/binding-venn-3.png" height="75%" width="75%" class="img-responsive"/>
                    </div>
                </section>
                
                <section data-transition="slide">
                    <h4>Strategy: use phage display to measure an unbiased sample of peptide targets</h4>
                    <div class="row" align="middle">
                        <img src="presentation-data/img/luke/phage-display.png" height="75%" width="75%" class="img-responsive"/>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>Human S100A5 and S100A6 both bind to a large number of partners, many of them shared</h4>
                    <img src="presentation-data/img/a5-a6-venn_0.png" height="100%" width="100%" />

                </section>

                <section data-transition="fade">
                    <h4>Human S100A5 and S100A6 both bind to a large number of partners, many of them shared</h4>
                    <img src="presentation-data/img/a5-a6-venn_1.png" height="100%" width="100%" />
                </section>
               
                <section>
                    <p>What about the ancestor?</p>
                </section>
 
                <section>
                    <h4>The ancestor is much more similar to human S100A6 than S100A5</h4>
                    <img src="presentation-data/img/triple-venn.png" height="50%" width="50%" />
                </section>

                <section>
                    <h4>Binding set of human S100A5 has both shrunk and shifted from the ancestral state</h4>
                    <img src="presentation-data/img/evol-change_0.png" height="100%" width="100%" />
                    <p class="fragment">Promiscuity to specificity on one lineage, ancestral promiscuity on the other.</p>
                </section>                       

                <section>
                    <h4>Remaining questions:</h4>
                    <ul>
                        <li>What are the genetic and biochemical bases for the altered specificity on the S100A5 lineage?</li>
                        <li>Do we see the same picture if we limit our binding set to "biological" peptides?</li>
                        <li>Can we mine new biological targets from our phylogenetically-informed, high-throughput data?</li>
                    </ul>                   
                </section>
 
                <section>
                    <h4>Conclusions:</h4>
                    <ul>
                        <li class="fragment">S100A5 and S100A6 have generally low peptide binding specificity</li>
                        <li class="fragment">Human S100A6 has preserved ancestral promiscuity</li>
                        <li class="fragment">Largest change has happened on the S100A5 lineage: increased specificity, altered set of targets</li>
                        <li class="fragment">A low-resolution experiment with known peptides gives a different answer than an unbiased, high-throughput measurement</li>
                    </ul>
                </section>


                <section>
                    <div class="row" style="color:gray">
                        <div class="col-xs-2">
                            <img src="presentation-data/img/people/luke-wheeler_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-10"><p>Luke Wheeler: Promiscuity and specificity in the evolution of S100 proteins</p></div>
                    </div>

                    <div class="row">
                        <div class="col-xs-2">
                            <img src="presentation-data/img/people/zach-sailer_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-10"><p>Zach Sailer: High-order epistasis in genotype-phenotype maps</p></div>
                    </div>

                </section>

                <section>
                    <h4>How does the distribution of function in sequence space shape evolutionary outcomes?</h4>
                </section>

                <section>
                    <p><em>Epistasis</em> (non-additivity between mutations) is a critical factor shaping evolution</p>
                    <h4 class="fragment">The case of oseltamivir resistance in H1N1 influenza</h4>
                    <ul> 
                        <li class="fragment">In 1991, the H274Y mutation to an influenza protein was shown to compromise viral fitness</li>
                        <li class="fragment">In 2008, H274Y entered the H1N1 population and conferred resistance to the drug oseltamivir</li>
                        <li class="fragment">Between 1991 and 2008, mutations accumulated that offset the deleterious effect of H274Y.</li>
                        <li class="fragment">Epistasis between these mutations opened a new evolutionary path</li>
                    </ul>
                    <br/>

                    <small>Bloom et al. (2010) <em>Science</em> 328:1272</small>
                </section>

                <section>
                    <p>Epistasis can have many effects:</p>
                    <ul>
                        <li class="fragment">Limits evolutionary trajectories (making evolution follow predictable trajectories). <br/><small>Weinreich et al (2006) <em>Science</em> 312(7):111</small></li>
                        <li class="fragment">Stochastically opens evolutionary trajectories (making evolution contingent on the past). <br/><small>Harms & Thornton (2014) <em>Nature</em> 512:203
</small></li>
                        <li class="fragment">"Entrenches" evolutionary trajectories (making evolution irreversible). <br/><small>Shah et al. (2015) <em>PNAS</em> 112(25):E3226</small></li>
                    </ul>           
                </section> 


                <section>
                    <p>But what if we expand sequence space?</p>
                    <img src="presentation-data/img/zach/wright_3.png" height="60%" width="60%"/><br/>
                    <small>Wright (1932) <em>Proceedings of the Sixth International Congress of Genetics</em> 356</small>
                </section>

                <section>
                    <p>Epistasis can be characterized through double mutant cycles</p>
                    <div class="row">
                        <div class="col-xs-12">
                            <img src="presentation-data/img/epistasis-models/dG-mutant-cycle.png"  />
                        </div>
                    </div>
                </section>
            
                <section>
                    <p>Epistasis can be characterized through double mutant cycles</p>
                    <div class="row">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-10">
                            <img src="presentation-data/img/epistasis-models/general-mutant-cycle.png" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                </section>

                <section>
                    <p>Larger sequence spaces could (in principle) give higher-order epistasis</p>
                    <div class="row">
                        <div class="col-xs-12">
                            <img src="presentation-data/img/epistasis-models/three-site.png" />
                        </div>
                    </div>
                </section>

                <section>
                    <p>What we (naively) expected:</p>
                    <img src="presentation-data/img/naive-decay.png" height="60%" width="60%"/>
                </section>

                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p>Does appreciable high-order epistasis exist?</p>
                        <p class="fragment">How would high-order alter evolutionary outcomes?</p>
                        <p class="fragment">Where does it come from?</p>
                    </div>
                </section>
                  
                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:red">Does appreciable high-order epistasis exist?</p>
                        <p style="color:gray">How would high-order alter evolutionary outcomes?</p>
                        <p style="color:gray">Where does it come from?</p>
                    </div>
                </section>

                <section>

                    <h4>Approach: measure high-order epistasis in published sequence spaces</h4>
                    <small>
                      <table>
                        <tr>
                            <td>Num. sites</td>
                            <td>Num. sequences</td>
                            <td>Genotype</td>
                            <td>Phenotype</td>
                            <td>Reference</td>
                        </tr>
                        <tr>
                            <td>5</td>
                            <td>$32\ (2^{5})$</td>
                            <td>Enzyme point mutations</td>
                            <td>Bacterial fitness</td>
                            <td>Weinreich (2006)</td>
                        </tr>
                        <tr>
                            <td>5</td>
                            <td>$128\ (2^{3} \times 4^{2})$</td>
                            <td>Protein/DNA point mutations</td>
                            <td>Protein/DNA affinity</td>
                            <td>Anderson (2015)</td>
                        </tr>
                        <tr>
                            <td>5</td>
                            <td>$32\ (2^{5})$</td>
                            <td>Enzyme point mutants</td>
                            <td>Bacterial fitness</td>
                            <td>Chou (2011)</td>
                        </tr>
                        <tr>
                            <td>5</td>
                            <td>$32\ (2^{5})$</td>
                            <td>Random genomic mutations</td>
                            <td>Fungual fitness</td>
                            <td>de Visser (2009)</td>
                        </tr>
                        <tr>
                            <td>6</td>
                            <td>$64\ (2^{6})$</td>
                            <td>Combinations of enzyme alleles</td>
                            <td>Yeast halploid growth rate</td>
                            <td>Hall (2004)</td>
                        </tr>
                        <tr>
                            <td>6</td>
                            <td>$64\ (2^{6})$</td>
                            <td>Combinations of enzyme alleles</td>
                            <td>Yeast diploid growth rate</td>
                            <td>Hall (2004)</td>
                        </tr>
                        <tr>
                            <td>6</td>
                            <td>$64\ (2^{6})$</td>
                            <td>Combinations of enzyme alleles</td>
                            <td>Yeast mating efficiency</td>
                            <td>Hall (2004)</td>
                        </tr>
                      </table>
                    </small>

                    <p class="fragment">Sequence spaces have very different genotypes and phenotypes</p>
                </section>
               
 
                <section data-transition="fade">
                    <div class="row">We detect statistically-significant high-order epistasis</div>
                    <div class="row" align="middle">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-10">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/wenriech-epistasis.png" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                </section>
                <section>

                    <p>Extract epistatic coefficients using the high-order model</p>
                    <div class="row">
                        <div class="col-xs-12">
                            <img src="presentation-data/img/epistasis-models/three-site.png" />
                        </div>
                    </div>
                    <p class="fragment">$\vec{\beta} = \mathbf{X}^{-1} \vec{f}$</p>

                </section>

                <section>
                    <p>High-order interactions are seen across all datasets investigated</p>
                    <div class="row">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-5">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/wenriech-epistasis.png" />
                        </div>
                        <div class="col-xs-5">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/anderson-dataset.png" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                    <div class="row">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-5">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/hall-1.png" />
                        </div>
                        <div class="col-xs-5">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/hall-2.png" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                            
                </section>

                <section>
                    <p>High order epistasis exists in all datasets</p>
                    <p>In general, the magnitude of epistasis does <em>not</em> decay with increasing order.</p> 
                </section>

                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:red">Does appreciable high-order epistasis exist?</p>
                        <ul class="fragment"><em><li style="color:#006400">Yes</li></em></ul>
                        <p style="color:gray">Does high-order epistasis matter?</p>
                        <p style="color:gray">Where does it come from?</p>
                    </div>
                </section>


                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:gray">Does appreciable high-order epistasis exist?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:red">Does high-order epistasis matter?</p>
                        <p style="color:gray">Where does it come from?</p>
                    </div>
                </section>


                <section>
                    <h4>Can high-order epistasis can shape evolutionary trajectories?</h4>
                    <p>Relative fitness of bacteria expressing $\beta$-lactamase variants</p>
                    <div class="row">
                        <div class="col-xs-6">
                            <img src="presentation-data/img/weinreich_network_0.png" />
                        </div>
                        <div class="col-xs-6">
                            <img class="fragment" src="presentation-data/img/weinreich_network_12.png" />
                        </div>
                    </div>
                    <small>Weinriech (2006) Science, 10.1126/science.1123539</small> 
                </section>
                
                <section data-transition="fade">
                    <div class="row">Knock out epistasis and see how it changes accessible trajectories</div>
                    <div class="row" align="middle">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-10">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/wenriech-epistasis.png" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                </section>

                <section data-transition="fade">
                    <div class="row">Knock out epistasis and see how it changes accessible trajectories</div>
                    <div class="row" align="middle">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-10">
                            <img class="img-responsive" src="presentation-data/img/experimental-epistasis/wenriech-epistasis_knockout.png" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                    
                </section>

                <section>
                    <h4>Removing high-order interactions expands the number of accessible trajectories</h4>
                    <div class="row">
                        <div class="col-xs-6">
                            <img src="presentation-data/img/weinreich_network_12.png" />
                        </div>
                        <div class="col-xs-6">
                            <img class="fragment" src="presentation-data/img/weinreich_network_30.png" />
                        </div>
                    </div>
                    <div class="row fragment">
                        <div class="col-xs-6">12</div>
                        <div class="col-xs-6">30</div>
                    </div>
                </section>

                <section>
                    <p>We can even see high-order epistasis just by staring at the sequence space</p>
                    <img src="presentation-data/img/weinreich-third-order.png" height="70%" width="70%"/>
                    <p class="fragment">High-order epistasis creates order-dependence in evolution.</p>
                </section>

                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:gray">Does appreciable high-order epistasis exist?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:red">Does high-order epistasis matter?</p>
                        <ul><em><li style="color:#006400">Yes</li></em></ul>
                        <p style="color:gray">Where does it come from?</p>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:gray">Does appreciable high-order epistasis exist?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:gray">Does high-order epistasis matter?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:red">Where does it come from?</p>
                    </div>
                </section>

                <!-- ENSEMBLE -->

                <section>
                    <h4>Do we expect this to be a general phenomenon?</h4>
                </section>

                <section>
                    <h4>Study high-order epistasis with simple protein computational model</h4>
                    <div class="row">
                        <div class="col-xs-1"></div>
                        <div class="col-xs-10">
                            <img src="presentation-data/img/lattice/lattice-models.png" class="img-responsive" />
                        </div>
                        <div class="col-xs-1"></div>
                    </div>
                    
                    <ul>
                        <li>Pairwise-only (second order) interactions</li>
                        <li>Can exhaustively sample thermodynamic ensembles across sequence space</li>
                    </ul>
    
                    <small>Implementation from: Bloom et al. (2006). <em>PNAS</em> 103(15):5869-5874</small>

                </section> 

                <section>
        
                    <h4>Strategy</h4>
                    <ol>
                        <li>Generate a 6-site ($2^{6} = 64$ protein) sequence space</li>
                        <li>Generate full thermodynamic ensembles for each sequence</li>
                        <li>Calculate $\Delta G_{native}$ for each ensemble</li>
                        <li>Fit the high-order epistasis model to these data</li>
                    </ol>

                </section>

                <section>
                    <p>Lattice model sequence space exhibits high-order epistasis</p>
                    <img src="presentation-data/img/lattice/lattice-model-epistasis.png" class="img-responsive"/>
                </section>

                <section>
                    <p><em>Where does this come from in a simple pairwise model?</em></p>
                    <p class="fragment">$$\Delta G_{A} = E_{A} - RTln \Big (Q - e^{-E_{A}/RT} \Big )$$</p>
                    <div class="fragment">
                        <p>For an ensemble with <em>only</em> states $A$ and $B$:</p>
                        <p>$$\Delta G_{A} = E_{A} - E_{B}$$</p>
                    </div>
                    <p class="fragment">A two-state model should not exhibit high-order epistasis</p>
 
                </section>

                <section>
                    <p>Two-state lattice model does not exhibit high-order epistasis</p>
                    <img src="presentation-data/img/lattice/lattice-model-with-only-two-states.png" class="img-responsive"/>
                </section>

                <section>
                    <p>If we add a third state, we expect high-order epistasis</p>
                    <p>$$\Delta G_{A} = E_{A} - RTln \Big (e^{-E_{B}/RT} + e^{-E_{C}/RT} \Big )$$</p>
 
                </section>
 
                <section>
                    <p>Three-state simulation: high-order epistasis</p>
                    <img src="presentation-data/img/lattice/three-way-lattice-epistasis.png" class="img-responsive"/>
                </section>

                <section>
                    <p>High-order epistasis arises from nonlinearity in system</p>
                    <ul>
                        <li class="fragment">Each mutation affects more than one conformation</li>
                        <li class="fragment">The system thus responds nonlinearly to mutations.</li>
                        <li class="fragment">The linear model accounts for this with high-order terms</li>
                    </ul>
                </section>
               
                <section>
                    <p>What about other sorts of nonlinearity?</p>
                    <p>Saturation:</p>
                    <img src="presentation-data/img/saturation-graph.png" />
                </section>

                <section>
                    <p>Simulate a completely additive dataset, but apply a saturation curve</p>
                    <img src="presentation-data/img/saturating-phenotype.png" height="70%" width="70%"/>
                </section>

                <section>
                    <p>Periodic, high-order epitasis appears</p>
                    <img src="presentation-data/img/saturating-phenotype_epistasis.png" />
                </section>
 
                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:gray">Does appreciable high-order epistasis exist?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:gray">Does high-order epistasis matter?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:red">Where does it come from?</p>
                        <ul><em><li style="color:#006400">From nonlinearity in the genotype-phenotype map</li></em></ul>
                    </div>
                </section>

                <section>
                    <p>Conclusions:</p>
                    <ul>
                        <li class="fragment">Biological phenotypes are generally non-linear</li>
                        <li class="fragment">Genotype space is fundamentally linear</li>
                        <li class="fragment">This mismatch means that high-order epistasis is "baked" into sequence space</li>
                        <li class="fragment">High-order epistasis leads to a profound "memory" in evolution: the effect of a mutation depends on all mutations that preceded it.</li>
                        <li class="fragment">Historical contingency, entrenchment, irreversibility, locally predictable evolutionary paths are the <em>rule</em>.</li>
                    </ul>
                </section>

                <section>
                    <h4>People</h4>
                    <div class="row" align="center">
                        <div class="col-xs-3"></div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/people/luke-wheeler_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/people/andrea-loes_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/people/zach-sailer_250x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/people/harman_369x369.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/people/jamie-bridgham_250x250.jpg" class="img-responsive"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/people/jim-prell_159x159.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-3"></div>
                    </div>
                    <div class="row" align="center">
                        <div class="col-xs-3"></div>
                        <div class="col-xs-1">
                            <small>Luke Wheeler</small>
                        </div>
                        <div class="col-xs-1">
                            <small>Andrea Loes</small>
                        </div>
                        <div class="col-xs-1">
                            <small>Zach Sailer</small>
                        </div>
                        <div class="col-xs-1">
                            <small>Joseph Harman</small>
                        </div>
                        <div class="col-xs-1">
                            <small>Jamie Bridgham</small>
                        </div>
                        <div class="col-xs-1">
                            <small>Prof. Jim Prell</small>
                        </div>
                        <div class="col-xs-3"></div>
                    </div>
                    <br/>
                    <small><em>Undergraduates: Bram Ricketts, Ran Shi, Hiranmayi Duvvuri, Sarina Pollat, Abigal Tamil, Jaclyn Smith</em></small>


                    <br/>
                    <h4>Funding</h4>
                    <div class="row">
                        <div class="col-xs-2"></div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/funding/sloan_250x250.png" class="img-responsive" height="50%"/>
                        </div>
                        <div class="col-xs-2">
                            <img src="presentation-data/img/funding/pew_500x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-2">
                            <img src="presentation-data/img/funding/aha_500x250.png" class="img-responsive"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/funding/ohsu_250x250.png" class="img-responsive" height="50%"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/funding/nih_250x250.png" class="img-responsive" height="50%"/>
                        </div>
                        <div class="col-xs-1">
                            <img src="presentation-data/img/funding/uo_250x250.png" class="img-responsive" height="50%"/>
                        </div>
                        <div class="col-xs-2"></div>
                    </div>
                </section>


                <!-- SCREWS UP LOW ORDER FIT -->
                <section>
                    <h4>What does high-order epistasis do to estimates of low-order terms?</h4>
                    <p class="fragment"><em>(can we just ignore it?)</em></p>

                    <p class="fragment" align="left">Strategy</p>
                    <ul>
                        <li class="fragment">Generate sequence space with 10th-order epistasis</li>
                        <li class="fragment">Regress low-ordered models against that space</li>
                        <li class="fragment">Measure two values:
                            <ul>
                                <li>$R^{2}$ between the model and the data</li>
                                <li>$R^{2}$ between the output and input model parameters</li>
                            <ul>
                        </li>
                    </ul>

                </section>
                
                <section data-transition="fade">

                    <p>Low-order models fit the phenotypes well...</p>
                    <p style="color:white">...but low order models scramble the input parameters!</p>
                    
                    <div class="row">
                        <div class="col-xs-10">
                            <img src="presentation-data/img/fit-instability/fit-vs-model-order_1.png" />
                        </div>
                        <div class="col-xs-2">
                            <div class="row">
                                <p style="color:red">$R^{2}$ model vs. data</p>
                            </div>
                            <div class="row">
                                <p style="color:white">$R^{2}$ input vs. output</p>
                            </div>
                        </div>
                    </div>

                </section>
    
                <section data-transition="fade">

                    <p>Low-order models fit the phenotypes well...</p>
                    <p>...but low order models scramble the input parameters!</p>
                    
                    <div class="row">
                        <div class="col-xs-10">
                            <img src="presentation-data/img/fit-instability/fit-vs-model-order_2.png" />
                        </div>
                        <div class="col-xs-2">
                            <div class="row">
                                <p style="color:red">$R^{2}$ model vs. data</p>
                            </div>
                            <div class="row">
                                <p style="color:blue">$R^{2}$ input vs. output</p>
                            </div>
                        </div>
                    </div>

                </section>   
 
                <section data-transition="fade">
                    <p>The apparent interaction between residue 1 and 2 fluctuates wildly at low order</p>
                    <img src="presentation-data/img/fit-instability/fluctuate_-1.png" class="img-responsive"/>
                </section>

                <section data-transition="fade">
                    <p>The apparent interaction between residue 1 and 2 fluctuates wildly at low order</p>
                    <img src="presentation-data/img/fit-instability/fluctuate_0.png" class="img-responsive"/>
                </section>

                <section data-transition="fade">
                    <p>The apparent interaction between residue 1 and 2 fluctuates wildly at low order</p>
                    <img src="presentation-data/img/fit-instability/fluctuate_1.png" class="img-responsive"/>
                </section>

                <section data-transition="fade">
                    <p>The apparent interaction between residue 1 and 2 fluctuates wildly at low order</p>
                    <img src="presentation-data/img/fit-instability/fluctuate_2.png" class="img-responsive"/>
                </section>

                <section data-transition="fade">
                    <p>The apparent interaction between residue 1 and 2 fluctuates wildly at low order</p>
                    <img src="presentation-data/img/fit-instability/fluctuate_3.png" class="img-responsive"/>
                </section>

                <section data-transition="fade">
                    <p>The apparent interaction between residue 1 and 2 fluctuates wildly at low order</p>
                    <img src="presentation-data/img/fit-instability/fluctuate_4.png" class="img-responsive"/>
                </section>

                <section>
                    <p>Parameter flipping occurs in experimental datasets as well</p>
                    <img src="presentation-data/img/fit-instability/blam-fluctuation.png" />


                </section>

                <section data-transition="fade">
                    <h4>Some questions</h4>
                    <div align="left">
                        <p style="color:gray">Does appreciable high-order epistasis exist?</p>
                        <ul><em><li style="color:gray">Yes</li></em></ul>
                        <p style="color:red">Does high-order epistasis matter?</p>
                        <ul><em>
                            <li style="color:#006400">Shapes evolutionary trajectories</li>
                            <li class="fragment" style="color:#006400">"Scrambles" low-order terms</li>
                        </em></ul>
                        <p style="color:gray">Where does it come from?</p>
                    </div>
                </section>



                <section data-transition="fade"> 
                    <h4>There is no simple trend of specificity for these proteins</h4>
                    <div class="row">
                        <div class="col-xs-2"></div>
                        <div class="col-xs-8">
                            <img src="presentation-data/img/phylogenetics-figs/anc-tree-1.png" class="img-responsive">
                        </div>
                        <div class="col-xs-2"></div>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>There is no simple trend of specificity for these proteins</h4>
                    <div class="row">
                        <div class="col-xs-2"></div>
                        <div class="col-xs-8">
                            <img src="presentation-data/img/phylogenetics-figs/anc-tree-2.png" class="img-responsive">
                        </div>
                        <div class="col-xs-2"></div>
                    </div>
                </section>
               
                <section>
                    <h4>Observations</h4>
                    <ul>
                        <li class="fragment">S100A5 and S100A6 bind to a huge variety of peptide targets</li>
                        <li class="fragment">In contrast, the number of peptides specific to S100A5 and S100A6 is small</li>
                        <li class="fragment">The evolution of their modern specificity was complicated</li>
                    </ul>
                    <h4>Unanswered questions:</h4>
                    <ul>
                        <li class="fragment">What happened to the total binding sets of AncA5, AncA6, and AncA5/A6?</li>
                        <li class="fragment">What is the physical mechanism by which historical mutations altered specificity?</li>
                    </ul>
                </section>


                <section>
                    <h4>Proteins only as specific as they need to be</h4>
                    <div class="row" align="middle">
                        <div class="col-xs-6">
                            <img src="presentation-data/img/parsing-env/small-people.jpg" class="img-responsive"/>
                        </div>
                        <div class="col-xs-6">
                            <img src="presentation-data/img/parsing-env/medium-people.jpg" class="img-responsive fragment"/>
                        </div>
                    </div>
                    <div class="row" align="middle">
                        <div class="col-xs-3"></div>
                        <div class="col-xs-6">
                            <img src="presentation-data/img/parsing-env/large-people.jpg" class="img-responsive fragment" height="75%" width="75%"/>
                        </div>
                        <div class="col-xs-3"></div>
                    </div>

                </section>
                <section>
                    <h4>Phage populations enrich in a calcium-dependent manner.</h4>
                    <br />
                    <div class="row" align="middle">
                        <div class="col-xs-12"><img src="presentation-data/img/phage-enrich-vs-rounds.png" class="img-responsive" height="60%" width="60%"/></div>
                    </div>
                </section>


                <section>
                    <h4>Questions</h4>
                    <div class="row">
                        <ul>
                            <li class="fragment">What are the sizes of the total binding sets of proteins?</li>
                            <li class="fragment">Do proteins become more specific over time?</li>
                            <li class="fragment">What, physically, drives these changes?</li>
                        </ul>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>S100 proteins are Ca2+ sensors</h4>
                    <div class="row">
                        <div class="col-xs-12">
                            <img src="presentation-data/img/s100-biology/biology-sketch-0.png" class="img-responsive"/>
                        </div>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>S100 proteins are Ca2+ sensors</h4>
                    <div class="row">
                        <div class="col-xs-12">
                            <img src="presentation-data/img/s100-biology/biology-sketch-1.png" class="img-responsive"/>
                        </div>
                    </div>
                </section>

                <section data-transition="fade">
                    <h4>S100 proteins are Ca2+ sensors</h4>
                    <div class="row">
                        <div class="col-xs-12">
                            <img src="presentation-data/img/s100-biology/biology-sketch-2.png" class="img-responsive"/>
                        </div>
                    </div>
                </section>
			</div>
            

		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>
    
		<script>

            var USE_LOCAL = false;

            if (USE_LOCAL) {
                var mathjax_location = 'MathJax-master/MathJax.js';
                var jquery_location = 'presentation-data/js/jquery-2.1.4.min.js';
                var bootstrap_location = 'presentation-data/js/bootstrap.min.js';
                var d3_location = 'presentation-data/js/d3.min.js';
            } else {
                var mathjax_location = 'https://cdn.mathjax.org/mathjax/latest/MathJax.js';
                var jquery_location = 'https://code.jquery.com/jquery-2.1.4.min.js';
                var bootstrap_location = 'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js';
                var d3_location = 'https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js';
            }

			// Full list of configuration options available at:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,

				transition: 'slide', // none/fade/slide/convex/concave/zoom

                // Add math, using local mathjax
                math: {
                    mathjax: mathjax_location,
                    config: 'TeX-AMS_HTML-full'  // See http://docs.mathjax.org/en/latest/config-files.html
                },

				// Optional reveal.js plugins
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true },
                    { src: 'plugin/math/math.js', async: true },    // mathjax
                    { src: jquery_location } ,
                    { src: bootstrap_location } ,
                    { src: d3_location } 
				]
			});



		</script>

	</body>
</html>
